### PR TITLE
Added /effect command

### PR DESCRIPTION
**Pull request by mathiascode**
This PR adds the vanilla /effect command to Core.

### CODE REVIEW

**Comment by mathiascode** [comment]
Anything I should change?

File: effect.lua [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	end
+	local RemoveEffects = function(OtherPlayer)
+		OtherPlayer:ClearEntityEffects()
+		if Split[2] ~= "@a" then
+			Player:SendMessageSuccess("Successfully removed effects from player \"" .. OtherPlayer:GetName() .. "\"")
+		end
+	end
+	if Split[3] ~= "clear" then
+		if Split[5] == nil then
+			Player:SendMessageInfo("Usage: " .. Split[1] .. " <player> <effect> [seconds] [amplifier] OR " .. Split[1] .. " <player> clear")
```
Comment (Seadragon91) [inline effect.lua]: Arguments in square brackets are optional? If yes then you have a wrong check above the usage `if Split[5] == nil then`

**Comment by mathiascode** [comment]
Fixed both issues.

**Comment by mathiascode** [comment]
Merge?

**Comment by madmaxoft** [comment]
Please rebase onto current master, so that the CI checks (that have been recently added) are run on your code, too.

**Comment by mathiascode** [comment]
Done.

**Comment by madmaxoft** [comment]
That's weird, I remember seeing that error, I thought I fixed it, and the CI test succeeds in current master. But I can't see the fix...

**Comment by madmaxoft** [comment]
Super-weird, some of my commits in the Checker repo have been lost. Luckily I was able to dig them out of my local repo and re-commit them. I've restarted the CI check for this PR with the new Checker.

**Comment by mathiascode** [comment]
Can someone merge this? Thanks.

**Comment by madmaxoft** [comment]
I guess we won't get any more reviews, so I'll merge.

**Pull request merged**